src/printer.js

```diff
@@ -1074,7 +1074,7 @@ function genericPrintNoParens(path, options, print) {
               n.selfClosing ? line : softline
             ])
           ),
-          n.selfClosing ? "/>" : ">"
+          n.selfClosing ? "/>" : ">",
```

Comment:
Hah, I should probably get in the habit of actually *using* prettier while I'm working on it...